### DISCUSSION TITLE
Extraction specific JSON keys in vector

### DISCUSSION THREAD

**Discussion by jintomv** [vectordotdev/vector#9023] [Q&A] [answered]
Can we extract specific fields of a JSON inside the "transform" block of vector ? Also we need to combine that fields in to yet another JSON.

**Comment by spencergilbert** [comment, marked as answer]
Hi, assuming all of these JSON objects are in the same event you can use a [remap transform](https://vector.dev/docs/reference/configuration/transforms/remap/) and [VRL](https://vector.dev/docs/reference/vrl/) to change the event's contents as needed. There's support for parsing JSON objects from strings, and also re-encoding if need be.
